code-challenges.js

```diff
@@ -6,16 +6,45 @@ var secretCodeWord1 = "lackadaisical"
 // Expected output: "l4ck4d41s1c4l"
 var secretCodeWord2 = "gobbledygook"
 // Expected output: "g0bbl3dyg00k"
-
-
-
-
-
+// creat a const that'll, take in a,e,i,o and return 0,4,3,1, 
+const codedWord = (str) => {
+    // use a .split on a new let function 
+    let strArray = str.split("") 
+    // creat a new var so you can do a .map on the function we just split
+    var newStr = strArray.map(value => {
+        //insert values you want to replace
+        if (value === "a"){
+        return value = 6
+    }else if (value === "e"){
+        return value = 3 
+    }else if (value === "i"){
+        return value = 1 
+    }else if (value === "o")
+        return value = 0
+     else {
+        return value
+        //return value is for anything that doesn't fall into the a,e,i,o 
+           }       
+              })
+              //use a .join so you can have them in the same places
+        return newStr.join("")
+            }
+console.log(codedWord(secretCodeWord1));
+console.log(codedWord(secretCodeWord2));
 // --------------------2) Create a function that takes in an array and returns all the words that contain the letter a.
 
 var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
+const favFruits = (array) => {
+   return array.filter(value => {
+       return value.includes("a") || value.includes("A")
```

Comment:
Consider a more dynamic way to do this so that you don't have to hard code the letter casing